.github/dependabot.yml

```diff
@@ -0,0 +1,29 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/support-frontend"
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "npm"
+    directory: "/support-workers/cloud-formation/src"
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: 'npm'
+    directory: '/cdk'
+    # The version of @aws-cdk/* libraries must match those from @guardian/cdk.
```

Comment:
This `cdk` configuration was copied from examples I've seen elsewhere in the Guardian organisation.